source/test/runbook_validator.test.ts

```diff
@@ -254,17 +251,18 @@ test.each(runbooks)('%s description has explanation', (runbook: RunbookTestHelpe
   expect(descriptionHasExplanation(description)).toBe(true);
 });
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
In general, I would prefer explicit disables to discourage use of `any`. There are certain contexts where `any` must be used more frequently, like when testing data of an unknown structure: cfn templates and ssm documents. It would probably be better if these were disabled for the file, in that case. I'll add an item to do that.